lib/gringotts/gateways/authorize_net.ex

```diff
@@ -94,7 +94,7 @@ defmodule Gringotts.Gateways.AuthorizeNet do
   @spec purchase(Float, CreditCard, Keyword) :: Tuple
   def purchase(amount, payment, opts) do
```

Comment:
set default value for opts as library user will not always provide this opts.

def purchase(amount, payment, opts \\ []) do